inference-engine/src/low_precision_transformations/src/low_precision.cpp

```diff
@@ -283,3 +284,24 @@ bool ngraph::pass::low_precision::LowPrecision::isFunctionQuantized(const std::s
     }
     return false;
 }
+
+bool ngraph::pass::low_precision::LowPrecision::isFQLevelsPresent(
+        const std::shared_ptr<const ngraph::Function>& function,
+        const std::set<size_t>& levels) {
+    std::vector<std::shared_ptr<ngraph::Node>> nodes = function->get_ops();
+    for (auto& node : nodes) {
+        for (size_t i = 0; i < node->inputs().size(); ++i) {
+            const auto fakeQuantize = as_type_ptr<ngraph::opset1::FakeQuantize>(node);
+            if (fakeQuantize != nullptr) {
+                if (levels.count(fakeQuantize->get_levels()) == 1) {
+                    return true;
+                }
+            }
+        }
```

Comment:
If you still like to use second iteration to check `FakeQuantize` operation, then don't need a loop over inputs here:
``` suggestion
        const auto fakeQuantize = as_type_ptr<ngraph::opset1::FakeQuantize>(node);
        if ((fakeQuantize != nullptr) && (levels.find(fakeQuantize->get_levels()) != levels.end())) {
            return true;
        }
```